package.json

```diff
@@ -59,6 +59,7 @@
     "postcss-preset-env": "^9.6.0",
     "prettier": "^3.3.3",
     "rollup-plugin-node-polyfills": "^0.2.1",
+    "sass": "^1.81.0",
```

Comment:
Ich glaube nicht, dass das Modul benötigt wird. Deine Erweiterungen in den `<style>` Tags verwenden keine SASS spezifische Syntax. Dadurch sollten auch alle Änderungen an der Datei `package-lock.json` hinfällig werden.
```suggestion
```